.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Description of what this PR fixes, steps to reproduce.
+
+```
+Codeblock to help reproduce issue.
+```
+
+**Checklist**
+[ ] Added JIRA issue in the message of one of your commits?
```

Comment:
Could we say "Did you add the JIRA issue in a commit message or as part of the branch name"?
I feel like having this strong rule can result in unnecessary extra work. For example, if there's no JIRA and then someone creates a JIRA, but you've already opened up a PR. In this case you'd have to rename, your branch, push and update the PR, and also delete the old branch locally and from the remote.

If you forgot to put the JIRA as part of a commit message you can just amend the commit, and force push which is a lot less effort.

What do you think?